### DISCUSSION TITLE
Err: "array contains too many dimensions" with arrays defined with curly braces

### DISCUSSION THREAD

**Discussion by onalv** [prisma/prisma#2952] [Q&A] [answered]
Hello, 

I'm getting the following error when trying to query a row which contains a column with data in this form:

`{{-23.0833329999999997,-70.4500000000000028},{-23.0308330000000012,-70.5020829999999989},{-22.874092000000001,-70.663337999999996}}`

**ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(Error { kind: FromSql(2), cause: Some("array contains too many dimensions") }) })**

When querying for that specific column is when I get the error. They are arrays of arrays but defined with curly braces. Is it that javascript doesn't recognize that as arrays? The array's length is 70 so I don't think the size is the problem. 

The prisma introspect marked the column as Float[] and the data type in the Postgres DB is defined as double precision[].

**Comment by ryands17** [comment, marked as answer]
Hey @onalv :wave: 
Could you share your database seed file in whch you added the table so I can introspect and test it?
Also which version of `@prisma/client` and `@prisma/cli` are you using?

**Reply by onalv** [reply]
Hey @ryands17 
thanks for the reply!

Unfortunately, I don't have a db seed file, this db was created some months ago. These are the versions I'm using:
"@prisma/cli": "^2.1.3"
"@prisma/client": "^2.1.3"

And here is an example of an example of data in this column (using random data but the same data structure and size). 
`{{-90.3800099999999997,-73.3633333333333398},{-90.3038003333333379,-73.6393899999999989},{-99.873399333333337,-73.660007999999996},{-99.9007073333333376,-73.7936699999999968},{-90.7993773333333339,-73.7996389999999970},{-90.9387983333333376,-73.8730739999999906},{-90.0399963333333376,-73.8997863333333336},{-90.0899963333333336,-73.8908693333333399},{-90.6698399996663909,-73.8967878637608670},{-90.7063373669860036,-73.8997073676033867},{-90.9399387737663379,-73.8096898366798779},{-93.3803970779397373,-73.8066397033633996},{-93.9673033686376637,-73.8086393373937936},{-93.3036666889668033,-73.8376837007066860},{-93.6333966873370936,-73.8336689783098678},{-93.7776907733773030,-73.8376699309880370},{-93.9677366666666663,-73.8636633333333376},{-96.7679966666666770,-73.8733600000000077},{-96.9837776666666773,-73.8879000000000067},{-96.6377799999999996,-73.9367366666666700},{-96.7783977677936336,-73.9998393677063387},{-96.8898696677730766,-73.9096360680337696},{-96.3679060973007969,-73.966007390693736},{-96.9096967880009736,-73.9707777366099736},{-96.3309389933767939,-73.9933678379687669},{-96.6769966389780379,-77.3373378993996987},{-96.7366083099793367,-77.3909983393939997},{-96.9779700000000989,-77.3373099999999960},{-97.3697680000000037,-77.3698000000000090},{-97.0706376666666667,-77.7778799999999996},{-97.3879700000000999,-77.7736866666666667},{-97.6796700000000076,-77.9708900000000070},{-97.7793600000000986,-77.9603900000000030},{-97.9676799999999988,-77.0936333333333390},{-98.7839666666666687,-77.0986900000000969},{-98.0796000000000039,-77.3303900000000077},{-98.3839769700368367,-77.6339730997660099},{-98.6679733333333336,-77.6676000000000987},{-98.7630366666666663,-77.6896600000000039},{-99.3638966666666667,-77.6839333333333396},{-99.9009966666666736,-77.6969366666666773},{-99.0808666666666667,-77.6993900000000063},{-99.6833333333333339,-77.793933333333333},{-99.6678700000000073,-77.7039000000000099},{-99.930900000000007,-77.7387799999999773},{-03.3897733333333376,-77.7630866666666679},{-03.3897733333333376,-77.7630866666666679},{-03.9686776939997337,-77.7673773697998039},{-03.3983987903083703,-77.7693089960763733},{-03.697387683787039,-77.7603733896879366},{-03.7669999396798067,-77.7636366663397393},{-03.9060708638767998,-77.7666339836896077},{-07.7368700000000039,-77.7666866666666636},{-07.9860866666666777,-77.7369900000000968},{-07.3687366666666668,-77.7966966666666733},{-07.6679600000000997,-77.7377799999999737},{-07.6809393333333373,-77.8399693333333336},{-07.8676097867073709,-77.8907809873906033},{-09.3798776809999600,-77.8077389308733607},{-09.7887970633679087,-77.8677639863788666},{-09.0666777339880979,-77.8669093639803967},{-09.6938037636037780,-77.879069397387030},{-09.6907080977376776,-77.8906937639333667},{-09.8673097639009778,-77.9377637696969933},{-00.3997063066798678,-77.9993036377676767},{-00.7983937939867933,-77.9060638997063666},{-00.0660369999999997,-77.9637633333333303},{-00.6736663333333303,-77.8766363333333373},{-00.683366999999999,-77.698777999999997}}
`

**Reply by ryands17** [reply]
@onalv This error is due to your field being a 2-D array. Unfortunately Prisma doesn't support multi-dimensional arrays at the moment. Could you create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into it further.

**Reply by onalv** [reply]
@ryands17 ok, well thank you for the help! I'll create the request.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
